client/quantum_serverless/core/tracing.py

```diff
@@ -74,7 +74,7 @@ def get_tracer(
     provider = TracerProvider(resource=resource)
     if agent_host is not None and agent_port is not None:
         otel_exporter = BatchSpanProcessor(
-            OTLPSpanExporter(endpoint=f"{agent_host}:{agent_port}")
+            OTLPSpanExporter(endpoint=f"{agent_host}:{agent_port}", insecure=True)
```

Comment:
Is it possible to configure the `insecure` variable through a environment variable?

Maybe I'm wrong but I think we can have some warnings from the security team. What do you think @psschwei @akihikokuroda ? I suppose that can be `True` for development and `False` for production.

Reply:
Yeah, that should be doable as an envvar, similar to what we do for `SERVICE_NAME` just above